Source/Shaders/GenerateShaders.py

```diff
@@ -200,7 +211,7 @@ def main():
 
                 with open(filename, "r") as dpd:
                     for line in dpd:
-                        if "#include" in line:
+                        if "#include \"" in line:
```

Comment:
I wonder what was the failure case?

Reply:
Right, I think we can remove this again. IIRC, what I did wrong was to run the script from the top-level build directory. This picked up C++ headers with `#include <vector>`.